README.md

```diff
@@ -43,32 +41,52 @@ preprocessors:
 `aliases`
 :   Mapping from aliases to Git repository URLs. Once defined here, an alias can be used to refer to the repository instead of its full URL.
 
+    >    **Note**
+    >
+    >    Aliases are available only within the legacy syntax of include statements (see below).
+
     For example, if you set this alias in the config:
 
         - includes:
-          aliases:
-            foo: https://github.com/boo/bar.git
-            baz: https://github.com/foo/far.git#develop
+            aliases:
+                foo: https://github.com/boo/bar.git
+                baz: https://github.com/foo/far.git#develop
 
-    you can include README.md file content from this repository using this syntax:
+    you can include the content of the files `doc.md` from these repositories using the following syntax:
 
         <<include>$foo$path/to/doc.md</include>
 
         <<include>$baz#master$path/to/doc.md</include> 
 
-    Note that in the second example we override the default revision.
+    Note that in the second example the default revision (`develop`) will be overridden with the custom one (`master`).
 
 ## Usage
 
-To include a document from your machine, put the path to it between `<<include>...</include>` tags:
+The preprocessors allows to use two variants of syntax of include statements.
+
+The **legacy** syntax is more simply and short, but it is less flexible, and there are no plans to extend it.
+
+The **new** syntax is more complicated and strict. This syntax is more suitable for complex cases, and it can be easily extended in the future.
+
+Both variants of syntax use the `<<include>...</include>` tags.
+
+If a local file path or a reference to a file in a remote Git repository is enclosed between these tags, the legacy syntax rules are used.
```

Comment:
If path is between the tags, it's the legacy syntax. If it's in the tag attributes (`src`, `repo_url`, and [others](#Attributes)), it's the new one.